health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/CampaignCard.js

```
@@ -14,47 +18,33 @@
  * campaign actions, such as setting up a campaign and viewing personal campaigns. The links are
  * filtered based on employee roles before being displayed in the EmployeeModuleCard component.
  */
+
 const CampaignCard = () => {
   if (!Digit.Utils.didEmployeeHasAtleastOneRole(Object.values(ROLES).flatMap((e) => e))) {
```

Biome

[error] Avoid unnecessary callback in flatMap call.

You can just use flat to flatten the array.
Safe fix: Replace unnecessary flatMap call to flat instead.


(lint/correctness/noFlatMapIdentity)